prototype/galore/galore_fused/triton/kernels/matmul.py

```diff
@@ -0,0 +1,383 @@
+import torch
```

Comment:
do we need a matmul kernel? This is something I figured torch.compile should be great at